### ISSUE TITLE
Commande volets

### ISSUE THREAD

**Issue by Tontonpat** [dubocr/homebridge-tahoma#276]
Bonjour Romain, j’utilise homebridge Alexa pour piloter mes équipements HomeKit. Depuis la nouvelle gestion des volets de ton plugin, la commande ouvre les volets dans Alexa règle mes volets en position MY.
Hors je règle la routine sur ouvert à 100%.
Sur tahoma:
![0257B664-055E-4B4A-9D7B-B6ABBDED057E](https://user-images.githubusercontent.com/49283285/112855972-a3fee680-90af-11eb-82fa-259d38f8cd42.jpeg)
Dans homebridge:
![3D59A924-6387-4859-85DB-8AE813077FD7](https://user-images.githubusercontent.com/49283285/112856260-e294a100-90af-11eb-966a-fc049b6974f4.jpeg)
Je ne comprends pas pourquoi ça fait ça.

**Comment by Tontonpat** [comment]
Je me suis connecté et mon numéro est 9950

**Comment by Dreiviertelblut** [comment]
There are unfortunately more issues now after installation of V2.1.20 with my RTS drives, as no MY position is working anymore as it used to with previous versions (V2.1.16). Tried to define "movementDuration" by putting 0 integer value to exclude possible "side effects", but does not make any difference. Seems, that this is not even recognized by the plugin. Trying to partially close the roller blinds, e.g. from 100% to 35% (MY) close them fully to 0%. Hombridge log also refers to "Close" and not "MY" command. The opposite direction from 0% to 35% open them to ~50%. 

As an alternative, I've also entered the measured movement Duration for one roller blind for testing purpose. In my case 46 secs. No change!

Rollback to V2.1.16 brought everything back to "normal".
